x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/types.ts

```diff
@@ -9,25 +9,16 @@ import type { BrowserFields } from '../../../../../common/search_strategy/index_
 import type { ColumnHeaderOptions } from '../../../../../common/types/timeline/columns';
 
 export type OnFieldSelected = (fieldId: string) => void;
-export type OnHideFieldBrowser = () => void;
 
 export interface FieldBrowserProps {
+  /** The timeline associated with this field browser */
+  timelineId: string;
   /** The timeline's current column headers */
   columnHeaders: ColumnHeaderOptions[];
   /** A map of categoryId -> metadata about the fields in that category */
   browserFields: BrowserFields;
-  /** The height of the field browser */
-  height: number;
   /** When true, this Fields Browser is being used as an "events viewer" */
   isEventViewer?: boolean;
-  /**
-   * Overrides the default behavior of the `FieldBrowser` to enable
-   * "selection" mode, where a field is selected by clicking a button
-   * instead of dragging it to the timeline
-   */
-  onFieldSelected?: OnFieldSelected;
-  /** The timeline associated with this field browser */
-  timelineId: string;
   /** The width of the field browser */
-  width: number;
+  width?: number;
```

Comment:
`height` property removed, `width` changed to optional and `onFieldSelected` no needed anymore